bedrock/firefox/views.py

```diff
@@ -606,18 +606,8 @@ def get_template_names(self):
             else:
                 template = 'firefox/whatsnew/index.html'
         elif channel == 'beta':
```

Comment:
I think can can perhaps remove this conditional, and let beta users fallback to whatever the default wnp is?